editoast/src/main.rs

```diff
@@ -667,16 +696,179 @@ async fn refresh_search_tables(
 mod tests {
     use super::*;
 
-    use crate::fixtures::tests::{db_pool, electrical_profile_set, TestFixture};
+    use crate::fixtures::tests::{
+        db_pool, electrical_profile_set, get_fast_rolling_stock, TestFixture,
+    };
     use diesel::sql_query;
     use diesel::sql_types::Text;
     use diesel_async::RunQueryDsl;
     use rand::distributions::Alphanumeric;
     use rand::{thread_rng, Rng};
     use rstest::rstest;
+    use serde::Serialize;
     use std::io::Write;
     use tempfile::NamedTempFile;
 
+    #[rstest]
+    async fn import_rolling_stock_ko_file_not_found(db_pool: Data<DbPool>) {
+        // GIVEN
+        let args = ImportRollingStockArgs {
+            rolling_stock_path: vec!["non/existing/railjson/file/location".into()],
+        };
+
+        // WHEN
+        let result = import_rolling_stock(args, db_pool).await;
+
+        // THEN
+        assert!(result.is_err())
+    }
+
+    #[rstest]
+    async fn import_non_electric_rs_without_startup_and_panto_values(db_pool: Data<DbPool>) {
+        // GIVEN
+        let rolling_stock_name =
+            "fast_rolling_stock_import_non_electric_rs_without_startup_and_panto_values";
+        let mut non_electric_rolling_stock = get_fast_rolling_stock(rolling_stock_name);
```

Comment:
You could wrap the created model in `TestFixture` so it would get cleaned up automatically, even if the test fails
I don't know if that's a good practice or not, what do you think @osrd-project/editoast-maintainers ?

Reply:
done